ktxspec.adoc

```diff
@@ -740,14 +740,30 @@ These 3 items combined form a _Data Format Descriptor_
 The full specification for this is found in the _Khronos Data
 Format Specification_ version 1.3 <<KDF13>>.
 
-If the _dfDescriptor_ describes an sRGB transfer function then
-`<<_vkFormat,vkFormat>>` must be one of the _SRGB_ formats.
-
 The _dfDescriptor_ is partially expanded here in order to provide
 sufficient information for a KTX2 file to be parsed without having to
 refer to <<KDF13>>. It consists of one or more _Descriptor Blocks_
 (dfDescriptorBlock).
 
+The following restrictions must be obeyed when setting the fields of the
+DFD and the image data must match the chosen values:
+
+* `color_model` must be `KHR_DF_MODEL_RGBSDA`, `KHR_DF_MODEL_YUVSDA`
+  or one of the block compressed color models listed in <<KDF13>> Section
+  5.6 or its successors, currently `KDF_DF_MODEL_BC1A` to
+  `KDF_DF_MODEL_PVRTC2`.
+* `color_primaries` may be any of the available values since conversion
+  to the display's primaries is relatively straightforward.
+  `KHR_DF_PRIMARIES_BT709/SRGB` is recommended for LDR images.
+  `KHR_DF_PRIMARIES_BT2020` is recommended for HDR images.
+* `transfer_function` must be `KHR_DF_TRANSFER_LINEAR` or
+  `KHR_DF_TRANSFER_SRGB` since these are the only ones supported by the
+  vast majority of graphics hardware and correct filtering and blending
+  without hardware support is difficult to impossible.
+* If `<<_vkFormat,vkFormat>>` is one of the `+*_SRGB{,_*}+` formats then
+  `transfer_function` must be `KHR_DF_TRANSFER_SRGB`. For all other
+  formats it must be `KHR_DF_TRANSFER_LINEAR`
```

Comment:
> For all other formats it must be `KHR_DF_TRANSFER_LINEAR`

Is compatible with Vulkan 1.1 multiplane video formats? Are we going to support them?

Reply:
Good question. We'll have to find out the answer. I do want to support at least YUV formats.

Reply:
For now I've excepted YUV formats and added a note that YUV support is not yet decided and the YUV section will be updated after a decision.